### ISSUE TITLE
"decrypt not recognized"

### ISSUE THREAD

**Issue by mcmatthevan** [matiffeder/stuff#1]

**Comment by alternistom** [comment]
I have the same issue :(

**Comment by Eyeball753** [comment]
In case anyone here is still interested, here is a working one: https://gbatemp.net/download/batch-cia-3ds-decryptor.35098/. It's actually how I found this repo. It's from the same dude.

**Comment by Framerr** [comment]
You dont have to download this github, look:
![no title](https://user-images.githubusercontent.com/26608145/209741043-e44a0b7f-230e-4d31-ae8d-eb0ed050c86f.png)

**Comment by ghost** [comment]
yeah, you just need to click download. but if you cant access the website, but you still have internet, you might need to find some workarounds.
